tests/test-infra/azure-aks.bicep

```diff
@@ -37,7 +37,7 @@ param diagStorageResourceId string = ''
 var osDiskSizeGB = 0
 
 // Version of Kubernetes
-var kubernetesVersion = '1.22.6'
```

Comment:
Can we keep it on 1.22.6 for now? That's what our partner teams are currently using in prod.

Reply:
These partner teams really should upgrade their Kubernetes version :)

I suppose we can keep this, but we constantly tend to fall behind in versions, so it's good to update this to the latest AKS version every now and then. We probably should have a separate approach for the partner teams going forward.